src/components/ArtifactDataMarkdown.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <p-markdown-renderer class="artifact-data-markdown" :text="data" />
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import { MarkdownArtifact } from '@/models'
+
+  const props = defineProps<{
+    artifact: MarkdownArtifact,
+  }>()
+
+  const data = computed<string>(() => props.artifact.data)
+</script>
+
+<style>
+.artifact-data-markdown { @apply
+  max-w-full
+  overflow-auto
+}
```

Comment:
Should the markdown renderer behave like this already?

Reply:
Perhaps but I didn't want to rely on that since it's technically not part of the component contract

Reply:
I'm just thinking generally block elements (and components) have height auto by default. No issue, just curious.